lib/src/supabase_auth.dart

```diff
@@ -288,6 +287,7 @@ extension GoTrueClientSignInProvider on GoTrueClient {
   ///
   ///   * <https://supabase.io/docs/guides/auth#third-party-logins>
   Future<bool> signInWithOAuth(
+    BuildContext context,
```

Comment:
Adding a required argument would be a breaking change, so could we make this an optional named argument?